lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/nested-journey-definitions.yaml

```diff
@@ -326,16 +321,14 @@ WEB_DL_OR_PASSPORT:
         back:
           targetState: MITIGATION_DL_CRI_DRIVING_LICENCE
         returnToRp:
-          targetJourney: FAILED
-          targetState: FAILED_SKIP_MESSAGE
+          exitEventToEmit: return-to-rp
```

Comment:
This one (and the equivalent for passport) now go to slightly different places, however:
- `FAILED_SKIP_MESSAGE` and `INELIGIBLE_SKIP_MESSAGE` are identical (call TICF and return to RP)
- It's arguably more consistent to describe this case as 'ineligible' anyway - you should only reach this page if you go to the DL CRI and then press the button to say you don't have a DVA or DVLA issued licence.